packages/compass-components/.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  root: true,
+  parser: '@typescript-eslint/parser',
+  plugins: [
+    '@typescript-eslint',
+    'jsx-a11y'
+  ],
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended'
```

Comment:
I think we can also add [react/recommended](https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules), [jsx-a11y/recommended](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y#rule-strictness-in-different-modes) and [jest/recommended](https://github.com/jest-community/eslint-plugin-jest#recommended) here right away

Reply:
nowadays we could even "tailor" stuff to the file extension:

```
".js" --> only js rule (no typescript parser)
".ts !.spec.ts" --> ts parser, + typescript/recommended + node/recommended 
".tsx" -->  react/recommended, jsx-a11y/recommended 
"spec.ts" -->  jest/recommended 
```

or something similar.. is a relatively new feature of eslint